project/legislation/legislatorslocation.py

```diff
@@ -0,0 +1,35 @@
+import Model from model
+
+def get_legislator_and_district(account):
+    """
+    This first sanity checks the zip, then makes sure it is in the USA.  If yes, continue.
+    This tries to pull from our local database, and failing that, from Propublica.
+    """
+    
+    location = account.zip_code
+
+    if ((location < 10000)||(location > 100000)):
```

Comment:
Consider moving this constraint to the model definition. E.g.

```py
postal_code = models.CharField(max_length=5)
```


Also, double check whether it is desirable, as the data coming from third party APIs is somewhat out of our control.